molgenis-core-ui/src/main/java/org/molgenis/ui/jobs/JobsController.java

```diff
@@ -105,4 +112,12 @@ public void runNow(@PathVariable("scheduledJobId") String scheduledJobId)
 	{
 		jobScheduler.runNow(scheduledJobId);
 	}
+
+	public String createJobExecutionHref(JobExecution jobExecution, int refreshTimeoutMillis) throws IOException
+	{
+		String jobHref = concatEntityHref("/api/v2", jobExecution.getEntityType().getId(), jobExecution.getIdValue());
+		String jobControllerURL = menuReaderService.getMenu().findMenuItemPath(ID);
+		return format("{0}/viewJob/?jobHref={1}&refreshTimeoutMillis={2}", jobControllerURL, jobHref,
+				refreshTimeoutMillis);
```

Comment:
Can you also apply this method in ```MappingServiceController``` and other places where we redirect to the job view?